analyzer-models/src/main/java/ffb/analyzer/models/espn/Player.java

```diff
@@ -65,7 +65,7 @@ public int getValue() {
     private List<Integer> eligibleSlots;
     private String firstName;
     private String lastName;
-    private int id;
+    private int playerId;
     private InjuryStatus injuryStatus;
     private int proTeamId;
```

Comment:
We were supposed to update proTeamID. Although, looking at the example data I can see that proTeamId goes from 1-33 so I'm pretty sure this aligns with football teams and not a unique player id (although there's actually 32 teams so something's off there - they may be considering a very old team or a value for free agency). Let's continue to track that separately - I'll update the issues we have.

I think creating the proTeamId enum is lower priority, so we can leave it as it's own issue and handle it later.